src/main/java/com/exasol/ExaConnectionAccessException.java

```diff
@@ -0,0 +1,18 @@
+package com.exasol;
+
+/**
+ * This exception indicates that the requested connection information could not
+ * be accessed, e.g. because the user has insufficient privileges or the connection
+ * does not exist.
+ */
+public class ExaConnectionAccessException extends Exception {
+    private static final long serialVersionUID = 1L;
+    public ExaConnectionAccessException(String msg) {
+        super("connection exception: "+msg);
+        this.msg = msg;
+    }
+    public String getMessage() {
```

Comment:
javadoc will probably demand javadoc comments for public methods.

Reply:
I wanted a 1:1 copy in this PR. I will fix code smells and JavaDoc in a later PR.